mscore/exportaudio.cpp

```diff
@@ -254,7 +257,12 @@ bool MuseScore::saveAudio(Score* score, const QString& name)
                   if ((mode & QIODevice::WriteOnly) == 0) {
                         return false;
                         }
-                  sf     = sf_open(qPrintable(filename), SFM_WRITE, &info);
```

Comment:
Actually I have the suspicion that this `qPrintable()` is the culprit already and alone.

Reply:
At first, I thought so too, replaced it with qUtf8Printable - not working.
Started searching, found that the problem is known https://github.com/erikd/libsndfile/issues/393
Especially for Windows, there is a separate function for opening files (need to do `#ifdef WINDOWS ...`)
I looked at what other opening functions are there, I saw that there is a function that takes a file descriptor and a decision came to my mind that I made, I think it is the most simple and reliable.

Reply:
It certainly looks sweet and simple.